app/controllers/stocks_controller.rb

```diff
@@ -1,23 +1,48 @@
 class StocksController < ApplicationController
   before_action :fetch_api, only: %i[index show create]
-
   def index; end
 
   def show
     @stock = @client.quote(params[:id].to_s)
+    @brokers = User.where(type: 'Broker')
```

Comment:
Did you use the `STI` setup?

Reply:
I did not use the STI setup for stocks since the columns of buyer stock and broker stock is fairly different, although they have some similarity.